include/deal.II/dofs/dof_handler.h

```diff
@@ -1674,6 +1674,55 @@ class DoFHandler : public Subscriptor
 #endif
 };
 
+
+/**
+ * A function that converts a Triangulation active cell iterator to an
```

Comment:
```suggestion
 * A function that converts a Triangulation active cell iterator to a
```
I would drop the "Triangulation" here because it is a DoFHandler iterator. It's not wrong, but slightly confusing.